src/dapps/ens-manager-dapp/components/manage/ManageUploadWebsite.vue

```diff
@@ -122,7 +126,12 @@ export default {
       this.uploadFile(e.target.files[0]);
     },
     setHash(val) {
-      this.ipfs = val;
+      this.error =
+        val === ''
```

Comment:
Can val ever be anything other than '' ?  Would it be better to use == to account for other falsy values?

Reply:
I think since the value is coming from text field it can only be '' but yea good observation, itll probably be better to use ==

Reply:
@rsteubs the input component returns null when cleared using the 'x' button within the component. I know @gesseekur changed this so this might now be necessary at all.